docs/api/gate.json

```diff
@@ -3,14 +3,22 @@
   "info": {
     "title": "Business Partner Data Management Gate",
     "description": "A gate for a member to share business partner data with CatenaX",
-    "version": "6.0.0"
+    "version": "6.0.2-SNAPSHOT"
```

Comment:
Version is updated now, need to update docs of api for  each service or we shall prefer to adapt the same during feature freeze?

Reply:
I will update the API docs with the newest SNAPSHOT version and then we can have a final look again

Reply:
Yeah, we can do that.